### ISSUE TITLE
clerk-expo now always requires installing expo-local-authentication

### ISSUE THREAD

**Issue by the-simian** [clerk/javascript#3950]
### Preliminary Checks

- [X] I have reviewed the documentation: https://clerk.com/docs

- [X] I have searched for existing issues: https://github.com/clerk/javascript/issues

- [X] I have not already reached out to Clerk support via email or Discord (if you have, no need to open an issue here)

- [X] This issue is not a question, general help request, or anything other than a bug report directly related to Clerk. Please ask questions in our Discord community: https://clerk.com/discord.

### Reproduction

N/A

### Publishable key

N/A

### Description

Documention requires updating: 

When using clerk-expo, you must now install `expo-local-authentication` or it will break on Android, *even if you're not using expo-local-authentication*

```ts
Android Bundling failed 23781ms node_modules/expo-router/entry.js (7369 modules)
Unable to resolve "expo-local-authentication" from "node_modules/@clerk/clerk-expo/dist/hooks/useLocalCredentials/useLocalCredentials.js"
```

This breaking change was introduced in https://github.com/clerk/javascript/commit/eb7cfae7c0d58f814a8db4dabf13f766e35c3e4e
on PR: https://github.com/clerk/javascript/pull/3663

Link to the related docs page
https://clerk-docs-git-prettier-fixes.clerkpreview.com/quickstarts/expo

Installing expo-local-authentication fixes the issue. This was a breaking change introduced in #3663

### Environment

```Shell
System:
    OS: Linux 5.15 Ubuntu 22.04.4 LTS 22.04.4 LTS (Jammy Jellyfish)
    CPU: (32) x64 13th Gen Intel(R) Core(TM) i9-13900K
    Memory: 54.17 GB / 62.68 GB
    Container: Yes
    Shell: 5.8.1 - /usr/bin/zsh
  Binaries:
    Node: 22.4.0 - ~/.nvm/versions/node/v22.4.0/bin/node
    npm: 10.8.1 - ~/.nvm/versions/node/v22.4.0/bin/npm
    pnpm: 8.15.1 - ~/.local/share/pnpm/pnpm
    bun: 1.1.18 - ~/.bun/bin/bun
```

**Comment by panteliselef** [comment]
Hey @the-simian Try following the instructions in #3954 and use this snapshot for a quick fix

```sh
npm i @clerk/clerk-expo@2.2.0-snapshot.v5f4676a --save-exact
```

**Comment by the-simian** [comment]
@panteliselef I installed the missing dependency to work past this for the time being, I've not tested the snapshot

**Comment by Gustavo-Kuze** [comment]
> Hey @the-simian Try following the instructions in #3954 and use this snapshot for a quick fix
> 
> ```shell
> npm i @clerk/clerk-expo@2.2.0-snapshot.v5f4676a --save-exact
> ```

I can confirm this snapshot fixes the issue.

**Comment by the-simian** [comment]
This seemed to also work for me, I'll use the snapshot for the time being

**Comment by LekoArts** [comment]
This has been fixed in @clerk/clerk-expo@2.2.0

**Comment by clerk-cookie** [comment]
This issue has been automatically locked since there has not been any recent activity after it was closed. Please open a new issue for related bugs.
